RecoLocalCalo/CaloTowersCreator/interface/CaloTowersCreationAlgo.h

```diff
@@ -353,7 +353,9 @@ class CaloTowersCreationAlgo {
   edm::Handle<EcalRecHitCollection> theEeHandle;
 
   //store merged depths for tower 28/29
-  std::vector<int> mergedDepths;
+  std::vector<int> mergedDepths, mergedDepthsOne;
```

Comment:
megedDepthsOne are the merged depths for tower 28/29 for the special RBX. I was not expecting any change in Run1. I shall just check if mergedDepths are different with the old and new implementation for Run 1.